espresso/src/main/java/com/elpassion/android/commons/espresso/recycler/RecyclerViewSizeMatcher.kt

```diff
@@ -15,9 +15,9 @@ fun containerHasChildCount(count: Int): Matcher<View> = object : TypeSafeMatcher
         return groupSize == count
     }
 
-    private fun getGroupCount(view: View): Int = when (view) {
-        is RecyclerView -> view.adapter.itemCount
-        is AdapterView<*> -> view.adapter.count
+    private fun getGroupCount(view: View): Int? = when (view) {
+        is RecyclerView -> view.adapter?.itemCount
+        is AdapterView<*> -> view.adapter?.count
         is ViewGroup -> view.childCount
         else -> throw IllegalArgumentException("Unknown view type")
     }
```

Comment:
I think we expect adapters to not be `null` and throw `IllegalStateException` otherwise.